source/codegen/metadata_mutation.py

```diff
@@ -59,6 +59,85 @@ def mark_mapped_enum_params(parameters, enums):
             del param['enum']
             param['mapped-enum'] = enum_name
 
+
+def get_grpc_type_from_ivi(type, is_array, driver_name_pascal):
+    add_repeated = is_array
+    if 'ViSession' in type:
+        type = 'nidevice_grpc.Session'
+    if 'ViBoolean' in type:
+        type = 'bool'
+    if 'ViReal64' in type:
+        type = 'double'
+    if 'ViInt32' in type:
+        type = 'sint32'
+    if 'ViConstString' in type:
+        type = 'string'
+    if 'ViString' in type:
+        type = 'string'
+    if 'ViRsrc' in type:
+        type = 'string'
+    if 'ViChar' in type:
+        if is_array == True:
```

Comment:
[Nit] While you're here can you remove the `== True` from the `if is_array` evaluations?